src/createListComponent.js

```diff
@@ -457,11 +458,15 @@ export default function createListComponent({
           return null;
         }
 
+        const isRtl = this.props.style && this.props.style.direction === 'rtl';
```

Comment:
I'm not sure about parsing the style in this way. Seems like RTL pages will likely have a `direction: rtl` on the level of the body rather than the list. Normally I'd say an explicit `direction` prop would be better, but that's already used in the list to differentiate between "horizontal" and "vertical". Hmm...

Reply:
In CSS this is called `direction` but that's already in use for lists. Ideally I would have maybe used "layout" or something but that ship has sailed, and I'm not really willing to do a major release just to rename this property.

There's a related HTML attribute called `dir` but that doesn't really follow the prop naming convention.

I briefly considered maybe using `direction` anyway and just expanding it to be "vertical", "horizontal", or "horizontal-rtl" but that feels awkward.

I am currently leaning toward an `isRTL` boolean attribute, even though...that's a bit weird too.

Reply:
Maybe `direction` can accept multiple values, e.g. "vertical ltr" (default), "horizontal ltr", "horizontal rtl", etc.

Reply:
Or maybe I should just add a new `layout` prop and deprecate `direction` (with a DEV warning) for now.

Reply:
In my use case I saw the usage of direction (CSS) in an isolated fashion (component biased). But in a real scenario is true that this can be affected by parent styles. Definitely the last approach seems the way to go.